.gitlab-ci.yml

```diff
@@ -223,11 +254,35 @@ cargo-check-benches:
   stage:                           test
   <<:                              *docker-env
   <<:                              *test-refs-no-trigger
+  <<:                              *cargo-check-benches-script
+  <<:                              *collect-artifacts
```

Comment:
`*collect-artifacts` names them like this `"${CI_JOB_NAME}_${CI_COMMIT_REF_NAME}"`. I think it's more future proof to add some git sha information, either in a dir name or in some metadata. Just in case someone debugs the regression, so it would be easier to compare the results from different commits.

Reply:
Resolved [here](https://github.com/paritytech/substrate/pull/8519/commits/7ed876a3e2dd43398832311b1b80f8aa3a3d8f89).